packages/neuron-wallet/src/services/live-cell-service.ts

```diff
@@ -16,17 +16,13 @@ export default class LiveCellService {
 
   constructor() {}
 
-  private async getLiveCellsByScript(
-    lock: Script | null,
-    type: Script | null,
-    data: string | null
-  ): Promise<LumosCell[]> {
+  private async getLiveCellsByScript(lock: Script | null, type: Script | null, data: string | null): Promise<Cell[]> {
     if (!lock && !type) {
       throw new Error('at least one parameter is required')
     }
 
-    const query: LumosCellQuery = { lock, type, data }
-    const liveCells: LumosCell[] = await queryIndexer(query)
+    const query = { lock, type, data } as QueryOptions
```

Comment:
```suggestion
    const query: QueryOptions = { lock, type, data } 
```

or 

```suggestion
    const query = { lock, type, data } satisfies QueryOptions
```

The "as" might cause an error in this context

Reply:
May be we can transfer `null` to `undefined`

Reply:
@homura  As described above, `as` is the only working option for now.

It's just a `null` or `defined` problem here, so using `as` is quite safe. And `getLiveCellsByScript` has good type check on its params.